src/translations/fr.json

```diff
@@ -1038,7 +1096,7 @@
       "image_categories": "personnages"
     },
     "person_fall": {
-      "event_types": "chute d'une personne"
+      "event_types": "déséquilibre"
```

Comment:
@momomaniac , @brunobesson  is it the good translation? 

Side note : The process is so cool, congrats @brunobesson